### ISSUE TITLE
Bi-directional photo library sync

### ISSUE THREAD

**Issue by sjpickup** [MobiusSync/MobiusSync#9]
Placeholder issue. Detail to be added. Please like (and ideally comment) if you want this feature.

**Comment by rokondo** [comment]
It would be really nice, even to select the folders properly and not using only the Mobius-Folder

**Comment by SimonPickup** [comment]
> It would be really nice, even to select the folders properly and not using only the Mobius-Folder

Created #59 to track this.

**Comment by mottbellow** [comment]
I updated the latest version, but I still don't see the option to sync photos.

**Comment by SimonPickup** [comment]
> I updated the latest version, but I still don't see the option to sync photos.

Correct, this issue is Open, meaning it's not implemented yet. ie Möbius Sync does not currently support photo library sync

**Comment by fy0z9b7q** [comment]
I would absolutely love Syncthing to keep photos bi-directionally in sync between my iPhone, iPad, and computers. Please consider giving this feature higher priority. I created a GitHub account specifically to make this comment, and would absolutely buy the app in a second for this feature! :-)

**Comment by SimonPickup** [comment]
I've been working on this and about to release a first POC on TestFlight. I'm excited! Stay tuned.

**Comment by SimonPickup** [comment]
Initial proof-of-concept support for **Send & Receive** Photo Collection folders is coming to TestFlight:

It has had very limited testing in very limited use cases:

- Create a new Album in iOS photos app, and add a local photo to it (currently Albums don't appear in the Möbius Sync photo collection picker until they contain at least one photo).
- Likely will not work for any generic collections, e.g. "All Photos", "Recents", "Favourites", "Videos" etc. _Not yet supported_
- To enable, switch **Folder Type** to **Send & Receive** under **Advanced** folder settings.
- When testing sending any photos/videos to iOS, be sure to sync copies (not your originals), in case of unexpected changes/deletions being synced back from iOS. Known problems:
  - File modification times and permissions are not yet synced by Möbius Sync; they WILL be modified when synced back from iOS.
  - If you delete and recreate a Syncthing folder association, Möbius Sync will forget which files files it imported and which have originated from the iOS Photo Colection, so ALL files will be renamed to the normal naming conventions, i.e. your photos will all be renamed on your remote device. (We are working on how to fix this but it is not simple.)
- Deletions on remote devices will be synced to iOS. You will be prompted to confirm deletion.
- Edits of imported files made on iOS will be synced out of iOS as a new file. This is how the iOS collection works internally.
- Edits of imported files made on remote devices will report an error. TODO
- Renaming of imported files made on remote devices may or may not work at this stage. TODO

**Comment by mottbellow** [comment]
666
